src/lib_ccx/ocr.c

```diff
@@ -526,7 +527,7 @@ char* ocr_bitmap(void* arg, png_color *palette,png_byte *alpha, unsigned char* i
 				else if(max==g_avg) h = 60 * ((b_avg - r_avg)/(max - min)) + 120;
 				else h = 60 * ((r_avg - g_avg)/(max - min)) + 240;
 
-				if(abs(h-h0)>50) // Color has changed
+				if(fabsf(h-h0)>50) // Color has changed
```

Comment:
Why? Do we have floats?

Reply:
`h` is a float (see a few lines above), and `h0` is declared as a float near the beginning of this block even though it's an integer constant.